doc/users/next_whats_new/donut_chart.rst

```diff
@@ -0,0 +1,29 @@
+Doughnut Charts 
+-----------------------------------
+
+A doughnut chart is essentially a pie chart with an area of the center cut out. It is one of the most aesthetically pleasing to look at charts that there is. In Matplotlib, it is apparent there there is no consolidated method of drawing a doughnut chart. With our changes we solve this problem.
```

Comment:
Let's not make judgment calls about the esthetics of donuts.

Reply:
Donuts need to be gustatory, not aestetic!